packages/components/src/BlankSlate/BlankSlate.js

```diff
@@ -0,0 +1,57 @@
+import { contextConnect, useContextSystem } from '@wp-g2/context';
+import { cx } from '@wp-g2/styles';
+import { useResizeAware } from '@wp-g2/utils';
+import React from 'react';
+
+import { Card, CardBody } from '../Card';
+import { Heading } from '../Heading';
+import { Icon } from '../Icon';
+import { Text } from '../Text';
+import { VStack } from '../VStack';
+import { cardBodyStyles, iconStyles } from './BlankSlate.styles';
```

Comment:
How I've been handling the imports of styles is to use:

```jsx
import * as styles from '...';
```

My thinking for that is that it's the same workflow as CSS Modules, which helps with familiarity.
This means that in the `BlankState.styles` file, the exports can just be `CardBody` or `Icon`.

And it can be used like `styles.CardBody` and `styles.Icon` within this file (just like CSS modules) :)